GUI/src/app/app.component.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
why body styles are here? and what these margins for?

Reply:
Yes, it's my fault, I forgot delete it